doc/doxygen/src/deprecated-modules.md

```diff
@@ -0,0 +1,21 @@
+Deprecated Modules    {#modules-deprecated}
+==================
```

Comment:
I would prefer, if this somehow could show up in the [`@deprecated` list](https://doc.riot-os.org/deprecated.html) so it is not easily forgotten by release managers. Can you try adding the `@deprecated` command here?

Reply:
The `@deprecated` command will link to some entity Doxygen is aware of such as function or type (or class). Since Doxygen is unaware of modules, I have no idea how to tell Doxygen to integrate it into the deprecation list.

Reply:
Modules are represented in Doxygen as groups, so you could link to the group.

Reply:
The more I think about this: Couldn't we just add the deprecated modules Makefiles add to doxygen and document the deprecation there in comments. We do a similar thing with some pseudomodules already.

Reply:
See https://github.com/RIOT-OS/RIOT/pull/18698.

Reply:
This would be my proposal to be in line with #18698 

```diff
diff --git a/makefiles/pseudomodules.inc.mk b/makefiles/pseudomodules.inc.mk
index 4059c1cab6..4f88138ca8 100644
--- a/makefiles/pseudomodules.inc.mk
+++ b/makefiles/pseudomodules.inc.mk
@@ -103,3 +103,10 @@ PSEUDOMODULES += gnrc_netif_mac
 PSEUDOMODULES += gnrc_netif_single
-PSEUDOMODULES += gnrc_netif_cmd_%
+##
+## @defgroup net_gnrc_netif_cmd_lora  gnrc_netif_cmd_lora
+## @ingroup shell_commands
+## @ingroup sys_gnrc_netif_lorawan
+## @{
+## @deprecated         Use `shell_cmd_gnrc_netif_lorawan` instead; will be removed after 2023.07 release.
+PSEUDOMODULES += gnrc_netif_cmd_lora
+## @}
 PSEUDOMODULES += gnrc_netif_dedup
```

Reply:
Ok, then let's just merge this and add the deprecation note either in #18698 or in a follow-up. However, nothing speaks against adding it here. But let's not bike-shed this any further :biking_woman:.

Reply:
See https://github.com/RIOT-OS/RIOT/pull/18706.